AIDevGallery/Utils/Ollama.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace AIDevGallery.Utils;
+
+internal record OllamaModel(string Name, string Tag, string Id, string Size, string Modified);
+
+internal class Ollama
```

Comment:
Change class name to something more meaningful.

Reply:
Done